ts/components/messages/MessageAgenda.tsx

```diff
@@ -252,82 +243,148 @@ const generateItemLayouts = (sections: Sections) => {
 
 const ItemSeparatorComponent = () => <View style={styles.itemSeparator} />;
 
-const FakeItemComponent = (
-  <View style={styles.itemEmptyWrapper}>
-    <Text style={styles.itemEmptyText}>{I18n.t("reminders.emptyMonth")}</Text>
+const MessageItemPlaceholder = (
+  <View style={[styles.itemLoadingContainer]}>
+    <View style={styles.itemLoadingHeaderWrapper}>
+      <View style={styles.itemLoadingHeaderCenter}>
+        <Placeholder.Paragraph
+          textSize={customVariables.fontSizeBase}
+          color={customVariables.shineColor}
+          lineNumber={2}
+          lineSpacing={5}
+          width="100%"
+          firstLineWidth="100%"
+          lastLineWidth="55%"
+          animate="shine"
+          onReady={false}
+        />
+      </View>
+    </View>
+
+    <View style={styles.itemLoadingContentWrapper}>
+      <View style={styles.itemLoadingContentCenter}>
+        <Placeholder.Line
+          textSize={customVariables.fontSizeBase}
+          color={customVariables.shineColor}
+          width="75%"
+          animate="shine"
+        />
+      </View>
+    </View>
   </View>
 );
 
+const SectionHeaderPlaceholder = (
+  <View style={styles.sectionHeaderWrapper}>
+    <View style={styles.sectionHeaderContent}>
+      <Placeholder.Line
```

Comment:
it seems similar to [this](https://github.com/pagopa/io-app/pull/1561/files#diff-ea9cc6ac235ffba1edf4270a064eed7aR266). Is it make sense for you to create a common item to avoid code duplication?